utils/runc/Makefile

```diff
@@ -1,15 +1,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=runc
-PKG_VERSION:=1.0.0-rc8
+PKG_VERSION:=1.0.0-rc8+3e425f80
 PKG_RELEASE:=1
```

Comment:
This versioning scheme is not correct.  Very likely the package manager cannot figure out which is newer among `1.0.0-rc8-1` and `1.0.0-rc8+<commit-id>-a-1`

Reply:
Changed to `1.0.0-rc8+<# commits since>-<commit id>`